care/facility/api/serializers/ambulance.py

```diff
@@ -35,6 +35,13 @@ def validate(self, obj):
             raise ValidationError("The ambulance must provide a price or be marked as free")
         return validated
 
+    def validate_vehicle_number(self, value):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5340830)